testsuite/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis-onboarding-experience/testsuite/execution_impl"
+	"github.com/kurtosis-tech/kurtosis-testsuite-api-lib/golang/lib/execution"
+	"github.com/sirupsen/logrus"
+	"os"
+)
+
+const (
+	successExitCode = 0
+	failureExitCode = 1
+)
+
+func main() {
+	// >>>>>>>>>>>>>>>>>>> REPLACE WITH YOUR OWN CONFIGURATOR <<<<<<<<<<<<<<<<<<<<<<<<
```

Comment:
You've actually already done this (with `MyTestsuiteConfigurator`), so you can get rid of these two comments